### PR TITLE
Port ShiftViT to Keras Core with TensorFlow backend

### PR DESCRIPTION
**Pull request by anas-rz**
Fix imports
In DropPath update shape computation: shape = (tf.shape(x)[0],) + (1,) * (len(x.shape) - 1)
Update loss computation in train_step and test_step
Remove TensorFlow addons dependency
[Colab Notebook](https://colab.research.google.com/drive/1-5CLMTHREpuU7QnRnKLKul9gIjn1MExI?usp=sharing)

### CODE REVIEW

**Comment by anas-rz** [comment]
New to OS.  @fchollet I am facing CLA fail issue whenever I create a PR.

File: examples/keras_io/tensorflow/vision/shiftvit.py [hunk 1]
```diff
@@ -0,0 +1,880 @@
...
+    """
+
+    def __init__(self, drop_path_prob, **kwargs):
+        super().__init__(**kwargs)
+        self.drop_path_prob = drop_path_prob
+
+    def call(self, x, training=False):
+        if training:
+            keep_prob = 1 - self.drop_path_prob
+            shape = (tf.shape(x)[0],) + (1,) * (len(x.shape) - 1)
```
Comment (anas-rz) [inline examples/keras_io/tensorflow/vision/shiftvit.py:266]: Changed Shape Computation

File: examples/keras_io/tensorflow/vision/shiftvit.py [hunk 2]
```diff
@@ -0,0 +1,880 @@
...
+        # Pass through the stages
+        x = projected_patches
+        for stage in self.stages:
+            x = stage(x, training=training)
+
+        # Get the logits.
+        logits = self.global_avg_pool(x)
+
+        # Calculate the loss and return it.
+        total_loss = self.compute_loss(data, labels, logits)
```
Comment (anas-rz) [inline examples/keras_io/tensorflow/vision/shiftvit.py:650]: Updated Loss Computation
